apps/openmw/CMakeLists.txt

```diff
@@ -141,7 +142,33 @@ target_link_libraries(openmw
     components
 )
 
-if (ANDROID)
+if (OPENMW_STATIC AND NOT OPENMW_NO_PKGCONFIG AND (NOT WIN32 OR MINGW))
```

Comment:
It might be a better idea to instead redirect pkg-config to use the right locations. For cross-compiling, I have this in the toolchain file:
```
# set env vars so that pkg-config will look in the appropriate directory for
# .pc files (as there seems to be no way to force using ${HOST}-pkg-config)
set(ENV{PKG_CONFIG_LIBDIR} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
set(ENV{PKG_CONFIG_PATH} "")
```
(where `CMAKE_INSTALL_PREFIX` is the cross-compiler's directory for headers and libs). That way, pkg-config will find the .pc files for packages installed for mingw.

Reply:
What do you mean exactly? pointing to the correct pkg-config isn't really the responsibility here. it's more that if your set on not using `pkg-config` you can turn it off. and Win32 to my knowledge has no pkg-config.

Setting the correct pkg-config dir and path should as you've shown, happen in the toolchain file

Reply:
> it's more that if your set on not using pkg-config you can turn it off. and Win32 to my knowledge has no pkg-config.

Hmm, well I could've sworn my MinGW came with x86_64-w64-mingw32-pkg-config. But either way, when installing libraries with MinGW they generally come with .pc files like for GCC. I don't see why you'd want to ignore them since they help get the correct paths and compiler flags.

> Setting the correct pkg-config dir and path should as you've shown, happen in the toolchain file

Right, which is why I'm not sure it's a good idea to ignore pkg-config with MinGW, since the toolchain file should set it up to work correctly.

Reply:
oooh, you're misreading the last expression :)

`NOT WIN32 OR MINGW`

it will use pkg-config things when it's not Win32 **or** uses MinGW.

So the only case where it's not doing pkg-config is when you're building for windows, but are not using MinGW

Reply:
Ah, sorry. My mistake.